.github/workflows/slow.yaml

```diff
@@ -14,6 +14,10 @@ on:
   push:
     branches: [ "auto" ]
 
+  # GitHub merge queue
+  merge_group:
+    branches: [ "master", "v7" ]
```

Comment:
Do we have to list v7 in a PR targeting "master" branch? v7-specific changes belong to v7 branch.

Reply:
Do we have to have Anubis private "auto" branch listed in code residing in "master" branch?

Reply:
I would not call "auto" branch _private_ -- it is a public Project branch that Anubis is using to stage Project PRs.

We list "auto" branch in push.branches for slow.yaml tests because we want slow.yaml tests to check staged commits (and staged commits are "pushed" to that "auto" branch). That desire applies to PRs targeting "master" branch (at least).

Reply:
You have answered your own question there.

Any official branch we are manually making PRs for is listed, because the github merge queue is potentially useful there. This setting also acts as a limiter to prevent that merge queue feature being used on non-listed branches (ie "auto" is not to be manually altered. Omitting it from this list prevents manually approving commits to change it).